master/usage/calicoctl/install.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Installing calicoctl
+no_canonical: true
+---
+
+## About installing calicoctl
+
+`calicoctl` allows you to create, read, update, and delete {{site.prodname}} objects 
+from the command line. You can run it on any host with network access to the 
+{{site.prodname}} datastore in either of the following formats. 
+
+- [Binary](#installing-calicoctl-as-a-binary): provides full functionality, including
+`node` commands for instances of `{{site.nodecontainer}}` on the same host.
+
+- [Container](#installing-calicoctl-as-a-container): provides less functionality than 
+the binary format, such as no use of the `node` commands.
+
+
+## Installing calicoctl as a binary
+
+1. Log into the host, open a terminal prompt, and navigate to the location where
+you want to install the binary. 
+
+   > **Tip**: Consider navigating to a location that's in your `PATH`. For example, 
+   > `/usr/local/bin/`.
+   {: .alert .alert-success}
+
+{% include {{page.version}}/ctl-binary-install.md %}
+
+1. Set the file to be executable.
```

Comment:
Any particular reason why this part (setting the file to executable, navigating to `/usr/local/bin` not a part of the included "ctl-binary-install" page? I think it should be as part of the included page and not here.

Reply:
Only the steps that vary are in the include. So, it is intentional @doublek

Reply:
Ah okay. In my mind I am thinking, the `chmod`'s and the `/usr/local/bin`-s are all part of installing the binary.